decksite/main.py

```diff
@@ -320,9 +323,10 @@ def community_guidelines() -> str:
 @APP.route('/rotation/<interestingness>/')
 @cached()
 def rotation(interestingness: Optional[str] = None) -> str:
-    rotation_query = request.args.get('rq')
-    _, cardnames = fetcher.search_scryfall(rotation_query, exhaustive=True) if rotation_query else (None, None)
-    view = Rotation(interestingness, rotation_query, cardnames)
+    query = request.args.get('fq')
+    if query is None:
+        query = ''
```

Comment:
If it’s easy it would be good to add this to /rotation/changes and /rotation/speculation too. That can definitely be in a follow up PR tho.

Reply:
Good idea, it shoudn't be hard. I'll make it in a follow up.